cypress/support/pages/change_password_page.js

```diff
@@ -32,11 +35,19 @@ class ChangePasswordPage {
     return cy.get('input[name="password_confirm"]:invalid');
   }
 
-  submitPasswordChange(password0, password1) {
+  typePasswordChange(password0, password1) {
     cy.get('input[name="password"]').type(password0);
     cy.get('input[name="password_confirm"]').type(password1);
+  }
+
+  submitPasswordChange(password0, password1) {
+    this.typePasswordChange(password0, password1);
     this.clickPasswordChange();
   }
+
+  stubBreachedCountEndpoint(response) {
```

Comment:
I'm not sure Page Objects are the right place for API based stubbing, they are really only to abstract out the page and Cypress' interaction with it. I think mocking should be either restricted to the actual test file or to a dedicated mocking helper file, much like the fixtures are off in their own files.

Reply:
👌  
moved in https://github.com/guardian/gateway/pull/123/commits/a0bc6ab714e0be6976cb358fb172040393d186d9